tests/integration/tier1/destructive/system-up-to-date/test_system_up_to_date.py

```diff
@@ -17,7 +17,11 @@ def test_skip_kernel_check(shell, convert2rhel):
     assert shell("mv /etc/yum.repos.d/* /tmp/s_backup/").returncode == 0
 
     # EUS version use hardcoded repos from c2r as well
-    if re.match(r"^(alma|rocky)-8\.6$", SYSTEM_RELEASE_ENV) or "centos-8-latest" in SYSTEM_RELEASE_ENV:
+    if (
+        re.match(r"^(alma|rocky)-8\.6$", SYSTEM_RELEASE_ENV)
+        or re.match(r"^(alma|rocky)-8\.8$", SYSTEM_RELEASE_ENV)
+        or "centos-8-latest" in SYSTEM_RELEASE_ENV
+    ):
```

Comment:
```suggestion
    if (
        re.match(r"^(alma|rocky)-8\.[68]$", SYSTEM_RELEASE_ENV)
        or "centos-8-latest" in SYSTEM_RELEASE_ENV
    ):
```
This should work as well

Reply:
Incorporated these changes in https://github.com/oamg/convert2rhel/pull/954/files